### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Frinkiac-7**
* Added list of API routes
  * Added links to client and API repos
  * Added links to client and API deployed links
  * Corrected style typo with 'Planned feature enhancements' section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,11 +3,36 @@ Clique is a geolocation-based file sharing client designed to help you explore y
...
+|--------|------------------------|-------------------------|
+| POST   | `/sign-up`             | `users#signup`          |
+| POST   | `/sign-in`             | `users#signin`          |
+| PATCH  | `/change-password`     | `users#changePassword`  |
+| DELETE | `/sign-out`            | `users#signout`         |
+| GET    | `/images`              | `images#index`          |
+| GET    | `/images/:id`          | `images#show`           |
+| POST   | `/images`              | `images#create`         |
+| PATCH  | `/images/:id`          | `images#update`         |
+| GET    | `/images/:id`          | `images#destroy`        |
```
Comment (ND56) [inline README.md:29]: Verb should be DELETE on line 29

File: README.md [hunk 2]
```diff
@@ -3,11 +3,36 @@ Clique is a geolocation-based file sharing client designed to help you explore y
...
+| POST   | `/images`              | `images#addComment`     |
```
Comment (ND56) [inline README.md:30]: Verb should be PATCH for lines 30-32

File: README.md (hunk 1 above)
Comment (Frinkiac-7) [reply README.md:29]: Ah, good catch.  Got burned by copy/paste.  Thanks, mate.

**Review by ND56 — COMMENTED** [review]
Rob, check out the comments I included in this review.

File: README.md (hunk 2 above)
Comment (Frinkiac-7) [reply README.md:30]: Thanks for the review, Nick.  Will revise and resubmit.

**Pull request closed without merging**